_includes/filter-controls.html

```diff
@@ -0,0 +1,94 @@
+            <label for="level-a">WCAG 2 Level A</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="level-aa" name="level" value="AA" checked>
+            <label for="level-aa">WCAG 2 Level AA</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="level-aaa" name="level" value="AAA">
+            <label for="level-aaa">WCAG 2 Level AAA</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-aria" name="level" value="aria" checked>
+            <label for="show-aria">WAI-ARIA</label>
+          </div>
+        </div>
+      </fieldset>
+      
+      <fieldset class="filter-group">
+        <legend class="filter-group-label">
+          Rule Status:
+          <a href="#about-rule-status" aria-label="about rule status" class="question">?</a>
+        </legend>
+        <div class="checkbox-group">
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-approved" name="status" value="approved" checked>
+            <label for="show-approved">Approved Rules</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-proposed" name="status" value="proposed" checked>
+            <label for="show-proposed">Proposed Rules</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-deprecated" name="status" value="deprecated">
+            <label for="show-deprecated">Deprecated Rules</label>
+          </div>
+        </div>
+      </fieldset>
+      
+      <fieldset class="filter-group">
+        <legend class="filter-group-label">
+          Implementation:
+          <a href="#about-implementations" aria-label="About implementations" class="question">?</a>
+        </legend>
+        <div class="checkbox-group">
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-manual" name="test-type" value="manual" checked>
+            <label for="show-manual">Manual</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-semiauto" name="test-type" value="semiauto" checked>
+            <label for="show-semiauto">Semi-Automatic</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-automated" name="test-type" value="automated" checked>
+            <label for="show-automated">Automated</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-linter" name="test-type" value="linter" checked>
+            <label for="show-linter">Linter</label>
+          </div>
+          <div class="checkbox-option">
+            <input type="checkbox" id="show-unimplemented" name="test-type" value="unimplemented" checked>
+            <label for="show-unimplemented">none</label>
```

Comment:
```suggestion
            <label for="show-unimplemented">None</label>
```

nit-pick, just for consistency